client/packages/purchasing/src/purchase_order/DetailView/DetailView.tsx

```diff
@@ -49,6 +60,8 @@ export const DetailViewInner = () => {
     isOpen,
   } = useEditModal<string | null>();
 
+  const lines = React.useMemo(() => data?.lines.nodes ?? [], [data]);
```

Comment:
Possibly overkill to memoise? It's a pretty trivial extraction.

Reply:
Ah, okay. Arguably those callbacks don't really need to be memo-ised either, but it's all good 👍